meta-balena-dunfell/recipes-core/systemd/systemd/0001-core-Don-t-redirect-stdio-to-null-when-running-in-co.patch

```diff
@@ -1,4 +1,4 @@
-From 1bf010678f1667d29132d8250328cbb7ea8059a5 Mon Sep 17 00:00:00 2001
+From 8fb27da9846d520358cd3b8ecc8d035b68d74086 Mon Sep 17 00:00:00 2001
```

Comment:
Please remove this and the commit it reverts. As this branch is still a work in progress there is no need to keep revert commits.

Reply:
I did a whole bunch of cleanup on the branch based on your suggestions, including dropping reverts.